docs/tools/vdb_table/data/couchbase.json

```diff
@@ -0,0 +1,181 @@
+{
+  "name": "Couchbase",
+  "links": {
+    "docs": "https://docs.couchbase.com/cloud/vector-search/vector-search.html",
+    "github": "https://github.com/couchbase",
+    "website": "https://www.couchbase.com/products/vector-search/",
+    "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/xx",
+    "poc_github": "https://github.com/1tylermitchell",
+    "slug": "couchbase"
+  },
+  "oss": {
+    "support": "partial",
+    "source_url": "https://www.couchbase.com/blog/couchbase-adopts-bsl-license/",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "https://www.couchbase.com/blog/couchbase-adopts-bsl-license/",
+    "comment": "Vector search currently only part of Enterprise and DBaaS editions"
+  },
+  "dev_languages": {
+    "value": [
+      "Python", "c++", "erlang", "go", "java"
+    ],
+    "source_url": "https://docs.couchbase.com/home/sdk.html",
+    "comment": "SDK client support: Python, C++, C, Go, Java, Kotlin, Node, PHP, Ruby, Scala, .Net"
+  },
+  "vector_launch_year": 2024,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "https://www.prnewswire.com/news-releases/couchbase-announces-new-features-to-accelerate-ai-powered-adaptive-applications-for-customers-302074941.html",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#hybrid",
+    "comment": ""
+  },
+  "facets": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/server/current/fts/fts-search-response-facets.html",
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/cloud/search/geo-search-ui.html",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/create-vector-search-index-ui.html",
+    "comment": ""
+  },
+  "bm25": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "full_text": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/c-sdk/current/concept-docs/full-text-search-overview.html",
+    "comment": ""
+  },
+  "embeddings_text": {
+    "support": "partial",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#large",
+    "comment": "via external embedding models"
+  },
+  "embeddings_image": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "via external embedding models"
+  },
+  "embeddings_structured": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "via external embedding models"
+  },
+  "rag": {
+    "support": "partial",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#large",
+    "comment": "via external LLMs"
```

Comment:
this refers to being able to generate the answers from the DB (via deep integration of LLM APIs. Have a look at the other DBs which have this marked as supported